src/Map.jsx

```diff
@@ -167,7 +167,9 @@ class Map extends Component {
         ...layers,
         this.state.showStops ? getStopMarkersLayer(route) : null,
         getRoutesLayer(geojson),
-        getVehicleMarkersLayer(route, info => this.displayVehicleInfo(info)),
+        // 270 is the angle for west bound trains, 90 for east
+        getVehicleMarkersLayer(route, info => this.displayVehicleInfo(info), 270),
+        getVehicleMarkersLayer(route, info => this.displayVehicleInfo(info), 90),
```

Comment:
Hey, so while this code works and gets the job done, I think it could be simpler and more extensible.
1. So it seems that our condition for choosing an icon is `If 0 <= heading <= 180, then BusEast else BusWest`
2. This is the first time that we're assigning different icons to buses. Would it make more sense if we modified getVehicleMarkersLayers to return a list of IconLayers for each group of buses? (where in this case we have two groups - EastBus and WestBus)

Reply:
Thanks, Eddy. That's a great suggestion - I'll work on that